README.md

```diff
@@ -154,10 +147,19 @@ kube-system   kube-proxy-wggdn           1/1       Running   0          3m
 kube-system   kube-scheduler-92zrj       1/1       Running   0          3m
 ```
 
-## Hyperkube version:
+### Stop it
 
-Example: `--hyperkube-version=1.9.3`
+Graceful stop it with:
+* SIGINT
+* SIGTERM
+* `--timeout`
+* `curl -XPOST 127.0.0.1:8989/stop`
+
+### Hyperkube versions
 
+`pupernetes` can start several Kubernetes versions with the flag `--hyperkube-version=1.9.3`.
```

Comment:
I think this is ambiguous, the user can chose among several versions of kubernetes for pupernetes to run, but pupernetes can only run one version of kubernetes at a time